### PR TITLE
Implementing integration tests for features.

### PR DESCRIPTION
**Pull request by proyectox123**

### CODE REVIEW

File: app/src/test/java/com/mho/training/TestDI.kt [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+import org.koin.core.module.Module
+import org.koin.dsl.module
+
+fun initMockedDi(vararg modules: Module) {
+    startKoin {
+        modules(listOf(mockedAppModule, mockedDataModule) + modules)
+    }
+}
+
+private val mockedAppModule = module {
```
Comment (jesusdmedinac) [inline app/src/test/java/com/mho/training/TestDI.kt:21]: You could create a dataSourceModule in order to separate the concerns.

File: app/src/test/java/com/mho/training/TestDI.kt [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    factory { CreditRepository(get()) }
+    factory { KeywordRepository(get()) }
+    factory { MovieRepository(get(), get(), get()) }
+    factory { PersonRepository(get()) }
+    factory { RegionRepository(get(), get()) }
+    factory { ReviewRepository(get()) }
+    factory { TrailerRepository(get()) }
+}
+
+val defaultFakeCredits = listOf(
```
Comment (jesusdmedinac) [inline app/src/test/java/com/mho/training/TestDI.kt]: Have you considered to put those fake objects into a sharetest module?

**Review by jesusdmedinac — APPROVED** [review]

File: app/src/test/java/com/mho/training/TestDI.kt (hunk 1 above)
Comment (proyectox123) [reply app/src/test/java/com/mho/training/TestDI.kt:21]: I'll check it.

File: app/src/test/java/com/mho/training/TestDI.kt (hunk 2 above)
Comment (proyectox123) [reply app/src/test/java/com/mho/training/TestDI.kt]: It's a great idea! I'll check it.

**Pull request merged**